magentov2/plugin/Indodana/PayLater/i18n/en_US.csv

```diff
@@ -0,0 +1 @@
+"FRAUD_MSG_LIST","Fraud Messages"
```

Comment:
Why do you need this?

Reply:
it is part of fraud hadle message. Because possibility of fraud entry is zero we assume that. should we remove this fraud class and what related to it ?